Nodejs/Product/InteractiveWindow/InteractiveWindow.csproj

```diff
@@ -252,4 +255,11 @@
   </PropertyGroup>
   <Import Project="$(BuildRoot)\Common\Product\ReplWindow\ReplWindow.proj" />
   <Import Project="..\ProjectAfter.settings" />
+  <Import Project="..\..\packages\SourceLink.Create.CommandLine.2.1.2\build\SourceLink.Create.CommandLine.targets" Condition="Exists('..\..\packages\SourceLink.Create.CommandLine.2.1.2\build\SourceLink.Create.CommandLine.targets')" />
```

Comment:
Extract a variable (property?) for the path so that the various instances stay in sync?